packages/app-mobile/components/screens/NoteTagsDialog.tsx

```
@@ -90,4 +91,4 @@ const NoteTagsDialog = connect((state: AppState) => {
 	};
 })(NoteTagsDialogComponent);
 
-export default NoteTagsDialog;
+export default NoteTagsDialog;
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)